tests/test_market_redis_connection.py

```diff
@@ -137,7 +137,7 @@ def test_accept_offer_calls_market_method_and_publishes_response(self):
                       seller="trade_seller", buyer="trade_buyer")
         self.market.accept_offer = MagicMock(return_value=trade)
         self.subscriber._accept_offer(payload)
-        sleep(0.01)
+        sleep(.1)
```

Comment:
Can you please explain this change? Why was it needed to increase the timeout?
Nit:I know it is the same and there is no general rule against it, but I personally prefer writing it like a human would write it on paper: `0.1` Thanks!

Reply:
Yes I changed the value because due to race condition and the fact the code is async, on my machine sometimes this fails.
I will change the format